ang/civicase/ContactCard.html

```diff
@@ -54,4 +64,60 @@
     </ul>
   </div>
   <!-- End - Additional Contact Dropdown -->
+  <script type="text/ng-template" id="contact-popover.html">
+    <div class="civicase__contact-popover">
+      <div class="civicase__contact-popover__header">
+        <h2>John Smith</h2>
+        <hr />
+      </div>
+      <div class="civicase__contact-popover__column">
+        <div class="civicase__contact-popover__detail-group">
```

Comment:
We are using `<strong>` tag for header text. Are we using  `civicase__contact-popover__detail-header` and `civicase__contact-popover__detail-value` for styling. If not can we get rid of these?